src/components/Plants/index.tsx

```diff
@@ -141,13 +142,11 @@ export default function PlantsDashboard(props: PlantsDashboardProps): JSX.Elemen
       <Grid item xs={12}>
         <PageSnackbar />
       </Grid>
-      <Box ref={contentRef} display='flex' flexGrow={1}>
+      <Box ref={contentRef}>
```

Comment:
This change undoes my last fix.. add it back?

Reply:
Oh, good catch, I must have been during the merge